cmd/search/http.go

```diff
@@ -31,6 +31,15 @@ type Match struct {
 	FileType     string      `json:"filename"`
 	Context      []string    `json:"context,omitempty"`
 	MoreLines    int         `json:"moreLines,omitempty"`
+	URL          string      `json:"url"`
+}
+
+type SearchResponseResult struct {
+	Matches []*Match `json:"matches,omitempty"`
+}
+type SearchResponse struct {
+	// SearchResults is a map of searchstring to search results that matched that search string
+	Results map[string]SearchResponseResult `json:"results,omitempty"`
```

Comment:
sorry, the omitempty comment in chat was meant for `URL` above, drop this one and move it up above

Reply:
done